master/buildbot/steps/master.py

```diff
@@ -38,18 +38,18 @@ class MasterShellCommand(BuildStep):
     description = 'Running'
     descriptionDone = 'Ran'
     descriptionSuffix = None
-    renderables = ['command', 'env', 'workdir']
+    renderables = BuildStep.renderables + ['command', 'env']
```

Comment:
I'm not 100% on this, but I dont think you have to pull in base-class renderables. I think it accumulates them across all bases automatically, right?